.github/workflows/test-gb-25.yml

```diff
@@ -53,8 +53,8 @@ jobs:
           - 'main'
           # - '0123456789abcdef0123456789abcdef01234567'
         reactant_commit:
-          - 'main'
-          # - "ap/updated_no_nan"
+          # - 'main'
```

Comment:
Keep main? This commit doesn't exist anymore

Reply:
testing that this pr doesn't mess up correctness, will revert before merging